src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -81,12 +82,17 @@ def create_subvolume(self, spec, size=None, namespace_isolated=True, mode=0o755)
         if size is not None:
             self.fs.setxattr(subvolpath, 'ceph.quota.max_bytes', str(size).encode('utf-8'), 0)
 
+        pool_set = False
```

Comment:
is this var required? this var is set when `pool` arg is valid (i.e., not `None`) -- just reuse `pool` instead.

Reply:
not required. I'll fix this